tensorflow/tools/docker/Dockerfile.gpu

```diff
@@ -1,9 +1,10 @@
-FROM nvidia/cuda:7.5-cudnn4-runtime
+FROM nvidia/cuda:7.5-cudnn4-devel
```

Comment:
I changed it to devel because otherwise there will be a couple of test failures. Some tests depend on devel headers and libraries. Maybe I can just blacklist those tests here?